akka-remote/src/main/scala/akka/remote/artery/Association.scala

```diff
@@ -228,17 +229,21 @@ private[akka] class Association(
   private def attachStreamRestart(streamName: String, streamCompleted: Future[Done], restart: Throwable ⇒ Unit): Unit = {
     implicit val ec = materializer.executionContext
     streamCompleted.onFailure {
+      case _ if transport.isShutdown     ⇒ // don't restart after shutdown
       case _: AbruptTerminationException ⇒ // ActorSystem shutdown
+      case cause: GaveUpSendingException ⇒
+        log.error(cause, "{} failed. Restarting it. {}", streamName, cause.getMessage)
```

Comment:
change this to debug level, or log at warning first time only, annoying when for example trying to join non-existing seed node
See also https://github.com/akka/akka/issues/20561#issuecomment-220302158